docs/contributor_guide/development_process.rst

```diff
@@ -101,24 +101,46 @@ Follow the steps below to create a pull request.
 3. Look at Fairlearn's issues on GitHub, specifically the ones marked `"help wanted" <https://github.com/fairlearn/fairlearn/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22>`_. Within this category we've marked issues with labels:
 
    - `"good first issue" <https://github.com/fairlearn/fairlearn/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+label%3A%22good+first+issue%22>`_ means issues are a good fit for first time contributors including people with no prior experience with coding or GitHub. This is an excellent way to get started!
-   
+
    - `"easy" <https://github.com/fairlearn/fairlearn/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+label%3A%22easy%22+>`_ means the issue is somewhat harder than a "good first issue", but still quite doable if you have prior experience or you're willing to spend a little bit of time.
-   
+
    - Neither of the two above: this means issues haven't been scoped out properly yet or are more difficult and likely won't be doable within a day or two. If you're still interested just let us know and we'll figure out a way! Once you find an issue you're interested in comment on the issue with any questions you may have and let us know that you'd like to do it. This helps us avoid duplication and we can help you quicker.
 
 4. Whenever questions come up don't hesitate to reach out (see :ref:`communication`). We're here to help!
 
-5. Clone the Fairlearn repository onto your machine using :code:`git clone https://github.com/fairlearn/fairlearn.git`
+5. Clone the Fairlearn repository onto your machine using
+
+.. code-block::
+
+ git clone https://github.com/fairlearn/fairlearn.git
```

Comment:
Contributors should be cloning from their own fork, not the upstream

similar to: https://scikit-learn.org/dev/developers/contributing.html#how-to-contribute

Reply:
A great catch, I just copied without rethinking there, as that was the link in the existing docs. I believe I'd like to change a few more things to look more like the scikit-learn version. 

Another thing is the introduction of the extension `sphinx-prompt` that scikit-learn's docs use. It's super convenient, but I didn't want to introduce another dependency just for a few edits. Now that I want to edit a larger block to look like the scikit-learn docs, I believe it might make more sense to include it. Any opinions on that? I could add it this PR together with the improvements.

Reply:
Yeah adding in this PR is fine. Dependencies for our CI / doc build are a no brainer, we can add them.

Reply:
Why is cloning from your fork better? I personally prefer cloning from the main repo and aliasing all forks by the corresponding usernames. Later in this doc we refer to a fork using the username, too, so everything else would need to be adjusted as well.

Maybe someone can explain the advantage of doing it another way?

Reply:
I also use the process that @adrinjalali suggested, I think it's a bit more straightforward for the users(especially less experienced people). I'm happy to change it all, I will go through all commands anyway as I introduce `sphinx-prompt`.

Reply:
@romanlutz the canonical way of cloning is to have `origin` as the fork, and `upstream` as the main repo

Reply:
Ok, whatever works for people. Just make sure to change every occurrence then because there may be a few of them in this doc.

Reply:
I think this is still WIP @TamaraAtanasoska ?

Reply:
Ah I see. Since this PR is not adding this, just formatting, I'm okay with it then, we can improve in a separate PR.